extensions/src/org/pentaho/platform/osgi/KarafInstance.java

```diff
@@ -139,7 +136,7 @@ private void assignInstanceNumber() {
     }
 
     instanceNumber = testInstance;
-
+    System.setProperty( "karaf.data", cachePath );
```

Comment:
@pamval 
Without this code if we try to start the 2 instance of kettle from one directory than one of them will fail because both try to use one cache folder.
This code add number for instance and will run both instance of kettle with different location of cache.

Which behavior should be in this case?